### PR TITLE
my final commit

### PR DESCRIPTION
**Pull request by Clementify**
I am done with my work please review @ChidiPatrick and @Promise-J.
Group-B.

### CODE REVIEW

**Review by promise-J — CHANGES_REQUESTED** [review]
Just one change to make. Please make the header nav links not to over-flow. 
Let it have a good font-size as it should please

File: index.html [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+                    African male wears can actually be worn by differnt people. 
+                    It's not meant for only African men.
+                </p>
+                <h3>$30</h3>
+            </div>
+            <button class="order">order now</button>
+        </div>
+        <div class="product card">
+            <div class="product-image">
+                <img src="./images/male2.jfif" alt="male2">
```
Comment (ChidiPatrick) [inline index.html]: Close you image tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="icon" href="./images/icon.jfifgi">
+    <title>FASHION ARENA</title>
+</head> 
+<body>
+    <header>
+        <div class="container">
+            <div class="left">
+            <img src="./images/logo main.png" alt="logo">
```
Comment (ChidiPatrick) [inline index.html]: close your tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+                    African male wears can actually be worn by differnt people. 
+                    It's not meant for only African men.
+                </p>
+                <h3>$25</h3>
+            </div>
+            <button class="order">order now</button>
+        </div>
+        <div class="product card">
+            <div class="product-image">
+                <img src="./images/male4.jfif" alt="male wears">
```
Comment (ChidiPatrick) [inline index.html]: close the tag

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Nice layout. Please close all your image tag properly, and make the header title responsive

**Review by ChidiPatrick — APPROVED** [review]
Great job! You can now merge

**Comment by Clementify** [comment]
> Just one change to make. Please make the header nav links not to over-flow. Let it have a good font-size as it should please

Done sir, Thank you.

**Pull request merged**